pico/services/src/main/java/io/helidon/pico/services/DefaultInjectionPlans.java

```diff
@@ -70,7 +71,13 @@ static Map<String, PicoInjectionPlan> createInjectionPlans(PicoServices picoServ
         }
 
         dependencies.allDependencies()
-                .forEach(dep -> accumulate(dep, result, picoServices, self, resolveIps, logger));
+                .forEach(dep -> {
+                    try {
+                        accumulate(dep, result, picoServices, self, resolveIps, logger);
```

Comment:
I observed in my work that when the maven-plugin was creating the injection plan, it would fail with the dependency being reported as unavailable, but it was not displaying the service provider that it failed on contextually. This is showing the "self" to contextually represent where in the processing it failed. The root error will still show what actually went wrong.